aten/src/THC/THCCachingAllocator.cpp

```diff
@@ -587,3 +587,50 @@ THC_API uint64_t THCCachingAllocator_maxMemoryCached(int device) {
   assertValidDevice(device);
   return caching_allocator.get_stats_for_device(device).max_amount_cached;
 }
+
+//
+// In CUDA IPC, sender sends a tensor to receiver, THCCaching_CUDAIpcDevptr
+// is called by receiver process to get access to the memory where the tensor
+// was built on sender process.
+//
+// CUDA IPC only allows sharing a big memory block associated with a IpcMemHandle,
```

Comment:
A general nit when writing documentation like this: if you write "IpcMemHandle" (capitalized specifically in this way), the implication is that this refers to an actual /name/ of some sort in the codebase. However, in this case, you're using this as an abbreviation for cudaIpcMemHandle_t. It's better to spell it out entirely, so that if someone greps for cudaIpcMemHandle_t they hit this code (they are probably not going to grep for IpcMemHandle).